e2e/nodebootstrapping_template.json

```diff
@@ -166,7 +166,6 @@
         "--max-pods": "110",
         "--network-plugin": "kubenet",
         "--node-status-update-frequency": "10s",
-        "--non-masquerade-cidr": "0.0.0.0/0",
```

Comment:
Thanks for updating this. I think that Windows should not use this flag since Windows uses ExceptionList. Please run all Windows test pipelines with this change to validate it.
<html>
<body>
<!--StartFragment-->

--non-masquerade-cidr string     Default: 10.0.0.0/8
--
  | Traffic to IPs outside this range will use IP masquerade. Set to '0.0.0.0/0' to never masquerade. (DEPRECATED: will be removed in a future version)

<!--EndFragment-->
</body>
</html>

Reply:
Should I also test it with the Windows VHD pipeline in this PR? Or just the Windows RP pipelines? Asking since the Windows pipelines are skipped for this PR.

Reply:
Vendored + tested in the AKS Windows E2E RP pipelines